src/arch/mips/reg/mips.rs

```diff
@@ -99,11 +136,12 @@ where
     fn gdb_deserialize(&mut self, bytes: &[u8]) -> Result<(), ()> {
         let ptrsize = core::mem::size_of::<U>();
 
-        // ensure bytes.chunks_exact(ptrsize) won't panic
-        if bytes.len() % ptrsize != 0 {
+        // Ensure bytes contains enough data for all 72 registers
+        if bytes.len() < ptrsize * 72 {
```

Comment:
These two checks aren't equivalent.

For example, if `bytes.len() == ptrsize * 72 + 1`, then `chunks_exact` will panic.

You can leave this condition in, but do not remove the other one.